Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -928,7 +928,7 @@ - (void)_updateScrollViewForTransaction:(const WebKit::RemoteLayerTreeTransactio
     WebKit::ScrollingTreeScrollingNodeDelegateIOS::updateScrollViewForOverscrollBehavior(_scrollView.get(), horizontalOverscrollBehavior, verticalOverscrollBehavior, WebKit::ScrollingTreeScrollingNodeDelegateIOS::AllowOverscrollToPreventScrollPropagation::No);
 
     bool hasDockedInputView = !CGRectIsEmpty(_inputViewBoundsInWindow);
-    bool isZoomed = layerTreeTransaction.pageScaleFactor() > layerTreeTransaction.initialScaleFactor();
+    bool isZoomed = WebKit::scalesAreEssentiallyEqual(layerTreeTransaction.pageScaleFactor(), layerTreeTransaction.initialScaleFactor()) ? false : layerTreeTransaction.pageScaleFactor() > layerTreeTransaction.initialScaleFactor();
```

Comment:
How about

```suggestion
    bool isZoomed = !WebKit::scalesAreEssentiallyEqual(layerTreeTransaction.pageScaleFactor(), layerTreeTransaction.initialScaleFactor()) && (layerTreeTransaction.pageScaleFactor() > layerTreeTransaction.initialScaleFactor());
```

Reply:
Was just about to leave this same comment; seems fine otherwise.